api/src/test/java/com/epam/pipeline/app/AclTestConfiguration.java

```diff
@@ -144,6 +146,12 @@
     @MockBean
     protected LogManager mockLogManager;
 
+    @MockBean
```

Comment:
Do we really need this mocks?

Reply:
You're right. This mock is redundant here. Removed.